saleor/plugins/manager.py

```diff
@@ -267,9 +267,12 @@ def calculate_checkout_shipping(
     def calculate_order_shipping(self, order: "Order") -> TaxedMoney:
         if not order.shipping_method:
             return zero_taxed_money(order.currency)
-        shipping_price = order.shipping_method.channel_listings.get(
+        channel_listing = order.shipping_method.channel_listings.filter(
             channel_id=order.channel_id
-        ).price
+        ).first()
+        if not channel_listing:
```

Comment:
Do we also want to raise an exception here? (If it is really must-have, I don't think that generic `Exception` is the best way for this). 
IMHO, I would recommend returning in that case `zero_taxed_money` as it is done as above. We can just simply assume that not having `channel_listing` is equal to not having shipping_method assigned to order. 
I believe that we are not able to finalize an order in case of missing `channel_listing` which I think it is enough  for forcing user to update a shipping.